doc/reference/configuration/configuration_reference.rst

```diff
@@ -2389,3 +2389,345 @@ The ``security`` section defines configuration parameters related to various sec
+
+    Limiting the queue size might be useful when a replica is trying to sync with a master and
+    reads new transactions faster than writing them to the WAL.
+
+    ..  NOTE::
+
+        You might consider increasing the ``wal.queue_max_size`` value in case of
+        large tuples (approximately one megabyte or larger).
+
+    |
+    | Type: integer
+    | Default: 16777216
+    | Environment variable: TT_WAL_QUEUE_MAX_SIZE
+
+..  _configuration_reference_wal_ext:
+
+wal.ext.*
+~~~~~~~~~
+
+..  admonition:: Enterprise Edition
+    :class: fact
+
+    Configuring ``wal.ext.*`` parameters is available in the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+
+This section describes options related to :ref:`WAL extensions <wal_extensions>`.
+The options allow you to add auxiliary information to each :ref:`write-ahead log <internals-wal>` record.
+
+Note that records with additional fields are :ref:`replicated <replication-architecture>` as follows:
+
+*   If a replica doesn't support the extended format configured on a master, auxiliary fields are skipped.
+*   If a replica and master have different configurations for WAL records, a master's configuration is ignored.
+
+..  _configuration_reference_wal_ext_new:
+
+..  confval:: wal.ext.new
+
+    Enable storing a new tuple for each :ref:`CRUD <box_space_examples>` operation performed.
+    The option is in effect for all spaces.
+    To adjust the option for specific spaces, use the :ref:`wal.ext.spaces <_configuration_reference_wal_ext_spaces>`
+    option.
+
+    |
+    | Type: boolean
+    | Default: false
+    | Environment variable: TT_WAL_EXT_NEW
+
+..  _configuration_reference_wal_ext_old:
+
+..  confval:: wal.ext.old
+
+    Enable storing an old tuple for each :ref:`CRUD <box_space_examples>` operation performed.
+    The option is in effect for all spaces.
+    To adjust the option for specific spaces, use the :ref:`wal.ext.spaces <_configuration_reference_wal_ext_spaces>`
+    option.
+
+    |
+    | Type: boolean
+    | Default: false
+    | Environment variable: TT_WAL_EXT_OLD
+
+..  _configuration_reference_wal_ext_spaces:
+
+..  confval:: wal.ext.spaces
+
+    Enable or disable storing an old and new tuple in the :ref:`WAL <internals-wal>` record
+    for a given space explicitly.
+    The configuration for specific spaces has priority over the configuration in the
+    :ref:`wal.ext.new <configuration_reference_wal_ext_new>` and :ref:`wal.ext.old <configuration_reference_wal_ext_old>`
+    options.
+
+    The option is a key-value pair:
+
+    *   The key is a space name (string).
+
+    *   The value is a table that includes two optional boolean options: ``old`` and ``new``.
+        The format and the default value of these options are described in ``wal.ext.old`` and ``wal.ext.new``.
+
+
+    Example:
+
+    ..  literalinclude:: /code_snippets/snippets/config/instances.enabled/persistence_wal/config.yaml
```

Comment:
Don't see the example

<img width="647" alt="image" src="https://github.com/tarantool/doc/assets/38073144/95c4ca0f-d351-439c-afc0-161c49a40e97">

Reply:
Fixed